app/routes/events/components/EventAdministrate/index.tsx

```diff
@@ -1,13 +1,14 @@
 import { Content } from 'app/components/Content';
 import { LoginPage } from 'app/components/LoginForm';
 import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
-import type { EventPool, EventAdministrate } from 'app/models';
+import type { EventPool, EventAdministrate, ID } from 'app/models';
```

Comment:
Also, use (and update) the store types. These types are deprecated

Reply:
WIP.

I'm trying to also do this in `Allergies.tsx`